front/app/containers/IdeasIndexPage/index.tsx

```diff
@@ -118,6 +119,12 @@ export default () => {
             <PageTitle>
               <FormattedMessage {...messages.inputsPageTitle} />
             </PageTitle>
+            <Box
+              position="absolute"
+              mt="-100px"
+              id="ideas-list-scroll-anchor"
+              aria-hidden={true}
+            />
             <IdeaCardsWithFiltersSidebar
```

Comment:
Because of the top nav bar, the scrollToElement doesn't scroll far enough up to view the element in it's entirety, even with block/inline: start.

I've added this anchor instead to make sure we scroll far enough up the page.

Reply:
There are some ways to achieve this (e.g., via window.scrollBy). But your solution seems clean enough to me! Let's just add a comment to the code with a condensed version of the above.